### DISCUSSION TITLE
Requesting HECVAT

### DISCUSSION THREAD

**Discussion by paballoCSMD** [audacity/audacity#5659] [General]
Hi There,

I was wondering if a developer would be able to contact me in regards to a HECVAT specifically an On-Premises form? https://library.educause.edu/resources/2020/4/higher-education-community-vendor-assessment-toolkit#tools

I am currently going through some academic red tape in order to get this piece of software approved. I need to make an honest effort contacting a representative (developer, admin, community manager). If there is no response, I would have to go through a different process to get it approved.

**Comment by petersampsonaudacity** [comment]
@LWinterberg  is this something you know anything about?  Or do you know who in Muse might?
